appveyor.yml

```diff
@@ -16,18 +15,20 @@ install:
   - "powershell ci-helpers/appveyor/install-miniconda.ps1"
   - "SET PATH=%PYTHON%;%PYTHON%\\Scripts;%PATH%"
   - "activate test"
+  - "conda install numpy=1.13.1 scipy=0.19.1 matplotlib pandas=0.20.3"
+  - "pip install flake8 pytest pytest-cov pytest-sugar pytest-faulthandler"
```

Comment:
do you need `pytest-sugar` ?

Reply:
it makes the output look nicer and easier for us to interpret